README.md

```diff
@@ -1,5 +1,21 @@
 # netbeans-l10n
 
-To build your own localization module, use the german module as template. Copy "local_de" and rename it (e. g. "locale_fr"). The locale needs to be available below netbeans-l10n-zip/src/. Then adapt `<property name="locale" value="de"/>` in build.xml.
+To build your own localization module, use the german module as template. The locale needs to be available below netbeans-l10n-zip/src/.
+## Steps to build your own localization module:
+* Copy "local_de" and rename it (e. g. "locale_fr").
+* Change the bundle dir in `src/org/apache/netbeans/l10n` from `de` to your locale (e. g. `fr`).
+* Change module name in  `src/org/apache/netbeans/l10n/Bundle.properties` to your locale (e. g. `French`).
+* In file `manifest.mf` change 
+	* line `OpenIDE-Module: org.apache.netbeans.l10n.de` to your locale (e. g. `org.apache.netbeans.l10n.fr`)
+	* line `OpenIDE-Module-Localizing-Bundle: org/apache/netbeans/l10n/de/Bundle.properties` to your locale (`org/apache/netbeans/l10n/fr/Bundle.properties`)
+* In file `build.xml` change 
+	* line `<project name="org.apache.netbeans.l10n.de" default="netbeans" basedir=".">` to your locale
+	* line `<description>Builds, tests, and runs the project org.apache.netbeans.l10n.de.</description>` to your locale
+	* line `<property name="locale" value="cde"/>` to your locale
+	* Then adapt `<property name="locale" value="de"/>`
```

Comment:
Is this the same as the line above? (We can remove this, right?)

Reply:
Yes, that is right. We can remove this duplicated line.

Reply:
I removed duplicated line.